### PR TITLE
Adding migrations for yellow ribbon

### PR DESCRIPTION
**Pull request by maneshkommidi**

### CODE REVIEW

**Comment by jkassemi** [comment]
@maneshkommidi - can you provide a link to the corresponding issue and some more context on how this table will be used?

File: db/schema.rb [hunk 1]
```diff
@@ -1202,6 +1202,48 @@
...
+  create_table "yellow_ribbons", force: :cascade do |t|
+    t.string   "facility_code"
+    t.string   "school_name_in_yr_database"
+    t.string   "school_name_in_weams"
+    t.string   "campus"
+    t.string   "street_address"
+    t.string   "city"
+    t.string   "state"
+    t.string   "zip"
+    t.string   "public_private"
```
Comment (jkassemi) [inline db/schema.rb]: Would like some more information on this column, seeing the following values:

* `- Please make a selection -`
* `NULL`
* `Private`
* `Private NP`
* `Private-NP`
* `Public`
* ``

File: db/schema.rb [hunk 2]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "degree_level"
+    t.string   "division_professional_school"
+    t.string   "contribution_amount"
```
Comment (jkassemi) [inline db/schema.rb]: Is this a nullable numeric?

File: db/schema.rb [hunk 3]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "updated_for_2011_2012"
```
Comment (jkassemi) [inline db/schema.rb]: All of the values in the source file are null or blank - is this supposed to be a boolean value?

File: db/schema.rb [hunk 4]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "missed_deadline"
```
Comment (jkassemi) [inline db/schema.rb]: would like to see this as a boolean, but I'm seeing a number of "- Please make a selection -" values. Can we get some clarity on this?

File: db/schema.rb [hunk 5]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "ineligible"
```
Comment (jkassemi) [inline db/schema.rb]: Same desire for a boolean here, but more of the "- Please make a selection -" values.

File: db/schema.rb [hunk 6]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "date_agreement_received"
```
Comment (jkassemi) [inline db/schema.rb]: Can this be a date column?

File: db/schema.rb [hunk 7]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "dat_yr_signed_by_yr_official"
```
Comment (jkassemi) [inline db/schema.rb]: Possibly a simple numeric, or ideally a date with a zeroed/default day/month.

File: db/schema.rb [hunk 8]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "amendment_date"
```
Comment (jkassemi) [inline db/schema.rb]: No non-null values in the source file yet, but I imagine this should be a date value

File: db/schema.rb [hunk 9]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "flight_school"
```
Comment (jkassemi) [inline db/schema.rb]: Requesting clarification here, too. Would like a boolean value, but we're seeing NO/YES/- Please make a selection - values.

File: db/schema.rb [hunk 10]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "date_confirmation_sent"
```
Comment (jkassemi) [inline db/schema.rb]: Date type preferred for this

File: db/schema.rb [hunk 11]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "consolidated_agreement"
```
Comment (jkassemi) [inline db/schema.rb]: Prefer boolean

File: db/schema.rb [hunk 12]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "new_school"
```
Comment (jkassemi) [inline db/schema.rb]: Prefer boolean

File: db/schema.rb [hunk 13]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "open_ended_agreement"
```
Comment (jkassemi) [inline db/schema.rb]: Prefer boolean

File: db/schema.rb [hunk 14]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "modified"
```
Comment (jkassemi) [inline db/schema.rb]: Prefer boolean

File: db/schema.rb [hunk 15]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "withdrawn"
```
Comment (jkassemi) [inline db/schema.rb]: Prefer boolean

File: db/schema.rb [hunk 16]
```diff
@@ -1202,6 +1202,48 @@
...
+    t.string   "sco_name"
+    t.string   "sco_telephone_number"
+    t.string   "sco_email_address"
+    t.string   "sfr_name"
+    t.string   "sfr_telephone_number"
+    t.string   "sfr_email_address"
+    t.string   "initials_yr_processor"
+    t.string   "year_of_yr_participation"
```
Comment (jkassemi) [inline db/schema.rb]: Would prefer a date column here, but there are values that specify multiple years, and more of the "- Please make a selection - " values, too. I'm guessing we should just keep this a string value for now, but more clarity required.

**Review by jkassemi — COMMENTED** [review]
Needs some feedback from the team that built out the source file, since there are some inconsistencies that I'd like to clarify before we set up the schema for import. 

It may be preferable to simply set up a table with columns for the data we need to use vs importing it all unless we can get some more consistency from the source.

**Comment by maneshkommidi** [comment]
@jkassemi Sure will review the CSV as per comments and address it Thanks.

**Comment by jkassemi** [comment]
Any updates on the field documentation?

**Pull request closed without merging**